### PR TITLE
fix #75377 consumer.huawei.com

### LINKED ISSUE #75377: consumer.huawei.com
### Issue URL (Annoyance)
 [https://consumer.huawei.com/ru/support/content/ru-ru00412920/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fconsumer.huawei.com%2Fru%2Fsupport%2Fcontent%2Fru-ru00412920%2F)
### Comment
Username: `@spok1`
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by wout-codes**
#75377

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+huawei.com##.support-bootom-cookie
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:94]: It's ok, but for future pull requests, don't add such rules in the first place. For example, add them a little lower.
<details><summary>Screenshot:</summary>

image

</details><br/>

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:94]: Wait a second, do you need this rule for `huawei.com`? Only here it's necessary `consumer.huawei.com`.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:94]: Ah yes. Not needed.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:94]: Why though? What conflicts?

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Our team have added rules to this place as new ones, so when contributors added them at the same place, we have conflicts.
That's why contributors don't have to add rules at the first place.

**Review by zloyden — APPROVED** [review]

**Pull request merged**